src/test/java/com/uwetrottmann/tmdb/services/TvEpisodesServiceTest.java

```diff
@@ -57,7 +103,7 @@ public void test_credits() {
         assertCrewCredits(credits.crew);
         assertCastCredits(credits.cast);
```

Comment:
Hier könnte man auch nach den Guest Stars testen. Bei den Methoden hier lässt sich auch evtl. viel Code teilen (Tests in statische Methoden auslagern).